src/LagoonCommands.php

```diff
@@ -18,6 +18,16 @@ class LagoonCommands extends DrushCommands implements SiteAliasManagerAwareInter
 
   use SiteAliasManagerAwareTrait;
 
+  /**
+   * Default ssh host, used for fallback
+   */
+  const DEFAULT_SSH_HOST = 'ssh.lagoon.amazeeio.cloud';
+
+  /**
+   * Default ssh port, used for fallback
+   */
+  const DEFAULT_SSH_PORT = 32222;
+
```

Comment:
we'd ideally make a function here to handle https://github.com/uselagoon/build-deploy-tool/pull/193 and fallback defaults to use in a chained null coalesce as per the api. We can probably assume that the last chance fallback doesn't need to be host and port configurable though?